content/guides/models/sweeps/add-w-and-b-to-your-code.md

```diff
@@ -113,7 +113,7 @@ def evaluate_one_epoch(epoch):
 
 
 def main():
-    run = wandb.init()
+    wandb.init()
```

Comment:
I believe that this change and the one on line 203 are in conflict with https://github.com/wandb/docs/pull/1146, which recently merged. Can you take a look and let me know what you think?

Reply:
Agreed. We might as well either use the recommended context manager expression or use **run.finish()** to ensure the completion of the run.

Reply:
I checked with @timoffex who did the recent edits, and lines 120-122 here should be updated according to this pattern:

```
lr = run.config["lr"]
bs = run.config["batch_size"]
epochs = run.config["epochs"]
```